pkg/cmd/cli/cmd/request_project.go

```diff
@@ -48,6 +43,41 @@ After your project is created it will become the default project in your config.
   %[1]s web-team-dev --display-name="Web Team Development" --description="Development project for the web team."`
 )
 
+// RequestProject next commands example.
+const (
+	requestProjectNewAppExample = `
```

Comment:
I get you probably did this because of the test, but repeating the string in the test is ok.